Frends.AzureBlobStorage.UploadBlob/Frends.AzureBlobStorage.UploadBlob.Tests/UnitTests.cs

```diff
@@ -422,6 +427,443 @@ public async Task UploadFileAsync_AppendAppendBlob()
         }

```

Comment:
https://github.com/FrendsPlatform/Frends.AzureBlobStorage/blob/83c90a429cd2d3618c23f15ae8fda28359c947cd/Frends.AzureBlobStorage.UploadBlob/Frends.AzureBlobStorage.UploadBlob.Tests/UnitTests.cs#L236
Please use `AreEqual` instead of `IsTrue`.

https://github.com/FrendsPlatform/Frends.AzureBlobStorage/blob/83c90a429cd2d3618c23f15ae8fda28359c947cd/Frends.AzureBlobStorage.UploadBlob/Frends.AzureBlobStorage.UploadBlob.Tests/UnitTests.cs#L270
Same here.